src/execution_algorithms/ltfb/regularized_evolution.cpp

```diff
@@ -204,25 +199,30 @@ void RegularizedEvolution::select_next(model& m,
     std::shuffle(sample_trainers.begin(),
                  sample_trainers.end(),
                  get_ltfb_generator());
+
+    // Print trainers selected in sample
+    std::cout << "Trainers in sample at step " << step << " -";
+    for (int i = 0; i < m_sample_size; i++)
+       std::cout << " " << sample_trainers[i];
+    std::cout << std::endl;
   }
   comm.world_broadcast(comm.get_world_master(),
                        sample_trainers.data(),
                        num_trainers);
 
-  // If rank within first S, send score , else score = 0
+  // If rank within sample, send score. Otherwise force score to 0 to indicate that trainer
+  // is not participating in tournament
```

Comment:
```suggestion
  // If rank within sample, send score. Otherwise force score to 0 to indicate that trainer
  // is not participating in tournament
  /// @todo Handle case where scores can be negative or where lower is better
```